src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/ReadInputArgumentCollection.java

```diff
@@ -12,6 +15,16 @@
  */
 public abstract class ReadInputArgumentCollection implements ArgumentCollectionDefinition {
     private static final long serialVersionUID = 1L;
+
+    @Argument(fullName = StandardArgumentDefinitions.VALIDATION_STRINGENCY_LONG_NAME,
+            shortName = StandardArgumentDefinitions.VALIDATION_STRINGENCY_SHORT_NAME,
+            doc = "Validation stringency for all SAM/BAM/CRAM files read by this program.  The default stringency value SILENT " +
+                    "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+                    "do not otherwise need to be decoded.",
+            common=true,
+            optional=true)
+    public ValidationStringency validationStringency = ValidationStringency.SILENT;
```

Comment:
Should we rename this to (eg.) `readValidationStringency`, since it only applies to reads? (note: applies to SRA as well, not just SAM/BAM/CRAM).